src/components/common/hoc/ContainerWrapper.spec.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+import ContainerWrapper from './ContainerWrapper';
+import ArticleComment from '../comments/ArticleComment';
+
+const initialState = {
+  auth: {
+    isAuthenticated: false,
+    user: {},
+  },
+};
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+const store = mockStore(initialState);
+
+describe('Container wrapper higher order component', () => {
+  const props = {
+    component: () => <ArticleComment />,
+    render: () => ({}),
+  };
+
+  it('should render the component without crashing', () => {
+    const wrapper = shallow(<ContainerWrapper {...props} />);
+
+    expect(wrapper).toMatchSnapshot();
```

Comment:
This assertion again?

Reply:
I did this assertion because it's a `HOC` and we really don't expect it to change since it simply just returns another component